docs/bridged_networks.md

```diff
@@ -0,0 +1,56 @@
+A wishlist of IRC networks to be bridged is being collected [in a github issue](https://github.com/matrix-org/matrix-appservice-irc/issues/208).
+
+| Network Name | Room alias format                         | Appservice user                    | Room for Support               | Operator                                  |
+|--------------|-------------------------------------------|------------------------------------|--------------------------------|-------------------------------------------|
+| darkfasel    | `#channame:darkfasel.net`                 | [`@IRC-Darkfasel:darkfasel.net`]   | [`#darkfasel:darkfasel.net`]   | [darkfasel](https://www.darkfasel.net/)   |
+| fc00         | `#fc00-irc_#channame:m.trnsz.com`         | [`@fc00ircmtx:m.trnsz.com`]        | None                           |                                           |
+| freenode[^1] | `#freenode_#channame:matrix.org`          | [`@appservice-irc:matrix.org`][^2] | [`#irc:matrix.org`]            | [Element Matrix Services]                 |
```

Comment:
Hmm, matrix.org would be better. All but one of the irc bridges are on actual matrix.org infra, and libera is on EMS but the hardware is loaned to matrix.org. In all cases, it's the matrix.org foundation who are maintaining those bridges :)

Reply:
Alright, changed to "Matrix.org" and linked to https://matrix.org :)